src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/courts/CourtsApiDelegateImpl.java

```diff
@@ -1,54 +1,40 @@
 package ca.bc.gov.open.jag.efilingapi.courts;
 
 import ca.bc.gov.open.jag.efilingapi.api.CourtsApiDelegate;
-import ca.bc.gov.open.jag.efilingapi.api.model.Address;
-import ca.bc.gov.open.jag.efilingapi.api.model.CourtLocation;
 import ca.bc.gov.open.jag.efilingapi.api.model.CourtLocations;
+import ca.bc.gov.open.jag.efilingapi.error.EfilingErrorBuilder;
+import ca.bc.gov.open.jag.efilingapi.error.ErrorResponse;
+import ca.bc.gov.open.jag.efilingceisapiclient.api.handler.ApiException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Service;
 
-import java.math.BigDecimal;
-import java.util.Arrays;
 
 @Service
 public class CourtsApiDelegateImpl implements CourtsApiDelegate {
+
+    Logger logger = LoggerFactory.getLogger(CourtsApiDelegateImpl.class);
+
+    private final CeisLookupAdapter ceisLookupAdapter;
+
+    public CourtsApiDelegateImpl(CeisLookupAdapter ceisLookupAdapter) {
+        this.ceisLookupAdapter = ceisLookupAdapter;
+    }
+
     @Override
     public ResponseEntity<CourtLocations> getCourtLocations(String courtLevel) {
-        return ResponseEntity.ok(buildMockData());
+        try {
+            logger.info("Request for court level recieved {}", courtLevel);
```

Comment:
is the a uid passed/generated through the request that we can log here? sorry if dumb question.